packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -103,6 +103,13 @@ class PlatformCameraUpdateZoomTo {
   final double zoom;
 }
 
+/// Pigeon representation of a CameraUpdateAnimationConfiguration.
+class PlatformCameraUpdateAnimationConfiguration {
```

Comment:
Pigeon interfaces are internal APIs, and can be changed at will, so there is no need to use parameter objects in general; the method can just take a durationMilliseconds parameter directly (same on iOS).